source/MRViewer/MRMenu.cpp

```diff
@@ -728,13 +728,24 @@ void Menu::draw_custom_plugins()
 
 void Menu::draw_helpers()
 {
-    if ( show_hotkeys_list_ )
+    if ( ImGui::GetIO().KeysDown[GLFW_KEY_F1] )
     {
-        ImGui::OpenPopup( "HotKeys" );
-        ImGui::BeginPopup( "HotKeys",
-                           ImGuiWindowFlags_AlwaysAutoResize | 
-                           ImGuiWindowFlags_NoResize
-        );
+        const auto style = ImGui::GetStyle();
+        const float hotkeysWindowWidth = 300 * menu_scaling();
+        size_t numLines = 3;
+        if ( shortcutManager_ )
+            numLines += shortcutManager_->getShortcutList().size();
+        
+        const float hotkeysWindowHeight = ( style.WindowPadding.y * 2 + numLines * ( ImGui::GetTextLineHeight() + style.ItemSpacing.y ) );
+
+        ImVec2 windowPos = ImGui::GetMousePos();
+        windowPos.x = std::min( windowPos.x, Viewer::instanceRef().window_width - hotkeysWindowWidth );
+        windowPos.y = std::min( windowPos.y, Viewer::instanceRef().window_height - hotkeysWindowHeight );
+
+        ImGui::SetNextWindowPos( windowPos, ImGuiCond_Always );
+        ImGui::SetNextWindowSize( ImVec2( hotkeysWindowWidth, hotkeysWindowHeight ) );
+        ImGui::Begin( "HotKeys", nullptr, ImGuiWindowFlags_AlwaysAutoResize | ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoFocusOnAppearing );
```

Comment:
no need to set up size, using `ImGuiWindowFlags_AlwaysAutoResize` flag
also use `ImGuiCond_Appearing` instead of `ImGuiCond_Always ` for `ImGui::SetNextWindowPos`